### PR TITLE
fix(popup.js) to save and load the customfiel values onCnage event

### PR DESCRIPTION
**Pull request by jenadiusnicholaus**
fix(popup.js) to save and load the customfiel values onCnage event

### CODE REVIEW

**Comment by joebordes** [comment]
formatting is incorrect. please read and apply [our guidelines](https://corebos.com/documentation/doku.php?id=en:devel:developmentguidelines)

File: popup.js [hunk 1]
```diff
@@ -78,6 +78,18 @@ document.getElementById('convertto').onchange=function (e) {
 				}
 			}
 			fset.innerHTML = fdef;
+			var customFields = [...document.querySelectorAll('.slds-input')]
```
Comment (joebordes) [inline popup.js]: this variable name is wrong. you are getting ALL the fields that have that class, the fixed fields also, so either change the name to something like `allFields` or set an attribute or a class on the custom fields and search on that to get only the custom fields.

File: popup.js [hunk 2]
```diff
@@ -78,6 +78,18 @@ document.getElementById('convertto').onchange=function (e) {
...
+			customFields.forEach((textfield)=> textfield.addEventListener('change',(e)=>{
+				chrome.storage.sync.set({'fieldValue':e.target.value});
+			}));
+			chrome.storage.sync.get("fieldValue",({fieldValue})=>{
+				if(fieldValue==undefined ){
```
Comment (joebordes) [inline popup.js]: this IF structure is too complex. it should be like this

```
if (fieldValue!=undefined) {
do whatever
}
```

the `return` you are using there adds no important information, and makes the code harder to read and maintain

**Comment by joebordes** [comment]
the commit message should be like this:

`feat(Popup) save and load the custom field values`

- it is a feat. the current functionality does not support saving the values, it isn't wrong, it is just the way it works, now we are adding new functionality: a feature
- we indicate what part of the application it affects (the Popup), not the file. if someone wants to know what files you have modified they can ask git
- we explain how we add the functionality

I think I would have written

`feat(Popup) persist the custom field values on Popup open`

just so you see another example

File: popup.js (hunk 1 above)
Comment (jenadiusnicholaus) [reply popup.js]: Yeah , i think, that is  is true, i am going to change it .

File: popup.js (hunk 2 above)
Comment (jenadiusnicholaus) [reply popup.js]: i get it

**Pull request closed without merging**